include/charstream.h

```diff
@@ -8,8 +8,12 @@
  * Transform input from a file descriptor into a stream of characters.
  */
 
+#define CHARSTREAM_EOF EOF
```

Comment:
```suggestion
#define CHARSTREAM_EOF (EOF)
```

Since EOF expands to a number as well, it's good to have parentheses around it

Reply:
Ignore charstream related stuff